engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -1,316 +1,459 @@
+  "biome-fog": "Mlha",
+  "biome-temperature": "Teplota",
+  "biome-humidity": "Vlhkost",
+  "biome-name": "Název",
+  "bloom-effect": "Rozostření",
+  "bobbing": "Pohyb hlavy",
+  "button-web-browser-confirmation-title": "Je vyžadováno potvrzení.",
+  "button-web-browser-confirmation-message": "Opravdu chceš otevřít toto url v prohlížeči?",
+  "camera-blur": "Rozmazání",
+  "camera-blur-max": "Maximální",
+  "camera-blur-normal": "Normální",
+  "camera-blur-off": "Vypnuto",
+  "camera-blur-some": "Trochu",
+  "camera-details-settings": "Nastavení kamery / grafického detailu",
+  "camera-film-grain": "Filmové zrnění",
+  "camera-setting": "Nastavení kamery",
+  "camera-setting-cinematic": "Cinematická",
+  "camera-setting-normal": "Normální",
+  "camera-setting-smooth": "Hladká",
+  "category-behavior": "Chování",
+  "category-general": "Obecné",
+  "category-interaction": "Interakce",
+  "category-inventory": "Inventář",
+  "category-mouse": "Myš",
+  "category-movement": "Pohyb",
+  "category-nui": "NUI",
+  "change-keybind-popup-message": "je již přiřazena. Přesto přiřadit?",
+  "change-keybind-popup-title": "Klávesa je už přiřazena",
+  "clamp-lighting": "Zamezit přesvětlení",
+  "cloud-shadows": "Stíny mraků",
+  "config": "Nastavit",
+  "connecting-to": "Připojuji se k",
+  "connection-failed": "Připojení selhalo!",
+  "continue": "Pokračovat",
+  "continue-pregeneration": "Pokračovat k Pregeneraci",
+  "continue-spawn-preview": "Pokračovat k náhledu Spawnu",
+  "continue-universe-setup": "Pokračovat k nastavení Vesmíru",
+  "could-not-connect-to-server": "Nepodařilo se připojit k serveru",
+  "crash-reporter": "Nahlásit selhání",
+  "create-game": "Vytvořit",
+  "create-game-title": "Vytvořit hru",
+  "create-preview": "Náhled",
+  "credits": "Autoři",
+  "custom-servers": "Vlastní",
+  "deactivate-module": "Deaktivovat",
+  "death-screen-title": "Jsi mrtvý",
+  "default-binding": "Výchozí přiřazení",
+  "delete-game": "Smazat",
+  "dev-tools-bt-editor": "Editor stromu chování",
+  "dev-tools-nui-editor": "NUI editor obrazovek",
+  "dev-tools-skin-editor": "NUI editor vzhledu",
+  "dev-tools-title": "Nástroje vývojáře",
+  "dialog-cancel": "Zrušit",
+  "dialog-no": "Ne",
+  "dialog-ok": "Ok",
+  "dialog-option": "Volba",
+  "dialog-yes": "Ano",
+  "disable-all-modules": "Deaktivovat vše",
+  "disable-launch-popup": "Zapamatovat a již se neptat",
+  "disable-rendering-class": "Vypnout",
+  "discord-presence": "Discord Rich Presence",
+  "download-module": "Stáhnout",
+  "downloading-server-list": "Načítání seznamu serverů ...",
+  "drop-item": "Zahodit předmět",
+  "edit-server": "Upravit",
+  "edit-server-title": "Přidat/Upravit server",
+  "enable-error-reporting": "Odesílat logy selhání na server",
+  "enable-telemetry": "Odesílat telemetrii na server",
+  "enable-rendering-class": "Zapnout",
+  "enter-username-message": "Ahoj a vítej v muliplayeru Terasology!\nPřeješ si aktualizovat tvé jméno?",
```

Comment:
```suggestion
  "enter-username-message": "Ahoj, a vítej ve hře pro více hráčů!\nPřeješ si aktualizovat svou přezdívku?",
```

Reply:
Multiplayer is probably the same case as spawn, but your suggestion is consistent with the translation already in the game, should review that a bit (maybe I even did that long time ago?)